tests/auth_tests/test_hashers.py

```diff
@@ -209,6 +214,53 @@ def setter(password):
         finally:
             hasher.rounds = old_rounds
 
+    @skipUnless(argon2, "argon2-cffi not installed")
```

Comment:
Not necessary for this. It could be done as a separate PR if you are so inclined.